env.sh

```diff
@@ -1,12 +1,13 @@
 NVCC=`which nvcc`
 export CUDA_ROOT=${NVCC%/*/*}
 
+CUDA_ENV="$PWD/.."
 ROOT="$PWD/.."
```

Comment:
I don't understand this change - why do we keep ROOT if it's not used anywhere?

Reply:
It is used at one place